### PR TITLE
Added sample for TF Flask tutorial

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Intending for this sample to live at https://cloud.devsite.corp.google.com/docs/terraform/terraform-in-google-cloud-quickstart, which will be replacing this community post: https://cloud.google.com/community/tutorials/getting-started-on-gcp-with-terraform

```release-note:none
```

### CODE REVIEW

**Review by slevenick — CHANGES_REQUESTED** [review]
It looks like this doesn't generate correctly. Not entirely sure why, but there are compilation errors due to this change

**Comment by slevenick** [comment]
Still failing with 
```
google-beta/resource_cgc_snippet_generated_test.go:25:29: expected '(', found '-'
?   	github.com/hashicorp/terraform-provider-google-beta	[no test files]
?   	github.com/hashicorp/terraform-provider-google-beta/version	[no test files]
```

**Comment by slevenick** [comment]
/gcbrun

Not sure why this isn't generating correctly

**Comment by betsy-lichtenberg** [comment]
> 

I thought I already removed public access protection in this commit:
https://github.com/GoogleCloudPlatform/magic-modules/pull/5839/commits/6c5c1c3705ff5fec38d47d56f9e50550a000f738

Removed again.

File: mmv1/products/cgc/terraform.yaml [hunk 1]
```diff
@@ -16,6 +16,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   Snippet: !ruby/object:Overrides::Terraform::ResourceOverride
     cgc_only: true
     examples:
+    # Compute
+      - !ruby/object:Provider::Terraform::Examples
+        name: "flask-google-cloud-quickstart"
```
Comment (c2thorn) [inline mmv1/products/cgc/terraform.yaml:21]: @betsy-lichtenberg the filename and reference to it here needs to be snake case. We use this name as-is in our code, and the hyphens cause build errors

**Comment by slevenick** [comment]
Looks like the test fails due to:

```
provider_test.go:309: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
(map[string]string) (len=2) {
(string) (len=10) "metadata.%": (string) (len=1) "1",
(string) (len=23) "metadata.startup-script": (string) (len=97) "sudo apt-get update; sudo apt-get install -yq build-essential python-pip rsync; pip install flask"
}
(map[string]string) (len=1) {
(string) (len=23) "metadata_startup_script": (string) (len=97) "sudo apt-get update; sudo apt-get install -yq build-essential python-pip rsync; pip install flask"
}
```

We will need to skip comparing this value in import by using `ignore_read_extra` in the example configuration. Likely on both metadata & metadata_startup_script

**Review by slevenick — APPROVED** [review]

**Pull request merged**